src/genie/libs/parser/iosxr/show_tacacs.py

```diff
@@ -0,0 +1,146 @@
+"""show_tacacs.py
+   IOSXR parsers for the following show commands:
+     *  show tacacs
+"""
+# python
+
+# Metaparser
+# python
+import re
+
+# Metaparser
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Optional
+
+
+# ==================================================
+# Schema for 'show tacacs'
+# ==================================================
+class ShowTacacsSchema(MetaParser):
+    """Schema for show tacacs"""
+
+    schema = {
+        'server': {
+            Any(): {
+                Optional('vrf'): str,
+                Optional('server_type'): str,
+                'opens': str,
+                'closes': str,
+                'aborts': str,
+                'errors': str,
+                'packets_in': str,
+                'packets_out': str,
+                'status': str,
+                'single_connect': str,
+                'family': str,
+            },
+        },
+    }
+
+
+# ==================================================
+# Parser for 'show tacacs'
+# ==================================================
+class ShowTacacs(ShowTacacsSchema):
+    """Parser for show tacacs"""
+
+    cli_command = ['show tacacs']
+
+    def cli(self, output=None):
+        if output is None:
+            cmd = self.cli_command[0]
+            out = self.device.execute(cmd)
+        else:
+            out = output
+
+        # initial dictionary
+        resultdict = dict()
+        serverdict = dict()
```

Comment:
Typically it's preferred to use literal declaration when possible, {} instead of dict(), but again I think that's an optional change.